docker/docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+version: '3.8'
+
+# Custom top-level network
+networks:
+  internal:
+  db:
+  web:
+
+services:
+  neo4j:
+    image: ${NEO4J_DOCKER_IMAGE:-neo4j:4.4}
+    hostname: neo4j
+    # Service-level network, which specifies the networks, from the list of the top-level networks (in this case only neo4j-internal), that the server will connect to.
+    # Adds a network alias (used in neo4j.conf when configuring the discovery members)
+    networks:
+      db:
+        aliases:
+          - neo4j-network
+    # The ports that will be accessible from outside the container - HTTP (7474) and Bolt (7687).
+    expose:
+      - 7687
```

Comment:
Should we add 7474 to this? Would be useful for admins trying to understand database issues by pulling up the web UI for queries and visualization.

Reply:
I will check that!